src/Groups/homomorphisms.jl

```diff
@@ -6,7 +6,12 @@ function Base.show(io::IO, x::GAPGroupHomomorphism)
 end
 
 function ==(f::GAPGroupHomomorphism{S,T}, g::GAPGroupHomomorphism{S,T}) where S where T
-   return f.map == g.map
+  return f.map == g.map
+end
+
+function Base.hash(f::GAPGroupHomomorphism{S,T}, h::UInt) where S where T
+  b = 0xdc777737af4c0c7b % UInt
+  return xor(hash(f.map, hash((S, T), h)), b)
```

Comment:
This is also problematic, because there is no hash for `f.map` only the generic one.

Perhaps I should in fact add a method `hash(::GapObj)` which throws an error to prevent such cases.

Reply:
I suggest keeping the version proposed here. The error will then be thrown by GAP.jl while trying to hash `f.map`.